client/me/security-checkup/style.scss

```diff
@@ -5,13 +5,18 @@
 }
 
 .security-checkup__nav-item {
+	&.card.is-compact {
+		padding-right: 24px;
+	}
+
 	span {
 		display: flex;
 		align-items: center;
 		> .security-checkup__nav-item-icon {
 			fill: var( --color-neutral-90 );
 			height: 32px;
 			margin-right: 16px;
+			min-width: 32px;
 			width: 32px;
```

Comment:
I think we can remove this `width: 32px;` rule now 🙂

Reply:
Good point! I did a quick check on Firefox and Chrome, and I think doing this is safe. (I'll verify that it's also OK on Safari via the live branch.)